### ISSUE TITLE
The Firefox browser is not working well and is not stable!

### ISSUE THREAD

**Issue by popoff-r** [Joshua-Riek/ubuntu-orange-pi5#93]
Firmware v.1.11 problems with firefox, when opening some
pages freezes and you have to force close the browser.
The Chromium browser works flawlessly.
Please help me to resolve this issue for me
firefox browser is very important!
I really like your firmware, thanks for the work done!

**Comment by Joshua-Riek** [comment]
I found the issue. Firefox is trying to use hardware acceleration and crashing. Go into your Firefox settings, and under Performance, uncheck "Use recommended performance settings"  then uncheck "Use hardware acceleration when available". However, sound in firefox is not working for me.

**Comment by popoff-r** [comment]
Thanks for the advice!
I did everything according to your recommendation, it helped !!!
Most of the sites have stopped hanging and open not badly.
Video from youtube plays with a resolution of only 360-480 pixel, the sound is present.
Very strange, on my OS Orangepi5_1.1.4_ubuntu_jammy_desktop_xfce_linux5.10.110
firefox browser works perfectly, 720-1080 video plays fine!

**Comment by Joshua-Riek** [comment]
I'm glad it's no longer hanging! The Orangepi5_1.1.4_ubuntu_jammy_desktop_xfce_linux5.10.110 OS does not have the proper video encoder for smooth 4k video playback using the GPU. I think Firefox sees the video encoder, but it's made only for Chromium which might create issues. I'll look into this a little more and see what I can find.

**Comment by Joshua-Riek** [comment]
Firefox can only play 480p videos right now. It requires H264 or VP9 video codec, which only Chromium supports. This is not something I can fix, but maybe the community will come up with something.

Source: https://support.mozilla.org/en-US/kb/firefox-4k-ultra-high-definition-youtube-videos#w_enable-vp9-in-advanced-preferences

**Comment by curyjorge98** [comment]
Hi All,

I'm testing v.1.11 and I am very happy with the rapid growth of development,

My WiFi dongle rtl8822bu is working pretty good.

Firefox is something I really want but I did not had success disabling Hardware accel. Hanging goes on.

popoff-r, this is for you, Testing Vivaldi Browser, it is working like a charm, playing youtube without hanging and more, "webgl aquarium" is playing better than Chromium, and I had to hold my chin up, (dont know this expression in english) it is playing 1000 fishs at 60 fps an without flickering (as Chromium does).

thanks Joshua for the great job.
